vkcs/networking/resource_port.go

```diff
@@ -95,16 +95,26 @@ func ResourceNetworkingPort() *schema.Resource {
 				ForceNew:    false,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				Set:         schema.HashString,
-				Description: "(Conflicts with `no_security_groups`) A list of security group IDs to apply to the port. The security groups must be specified by ID and not name (as opposed to how they are configured with the Compute Instance).",
+				Computed:    true,
+				Description: "(Conflicts with `no_security_groups`) A list of security group IDs to apply to the port. The security groups must be specified by ID and not name. External changes are not displayed by default, to see them you need to set the `full_security_groups_control` argument to true.",
```

Comment:
could we use schema feature for conflict detection?

Reply:
no( We need a case with sg_ids = [] and no_sgs = true